p2p/src/main/java/bisq/network/p2p/peers/PeerManager.java

```diff
@@ -176,7 +176,7 @@ public void readPersisted() {
         PeerList persistedPeerList = storage.initAndGetPersistedWithFileName("PeerList", 1000);
         if (persistedPeerList != null) {
             long peersWithNoCapabilitiesSet = persistedPeerList.getList().stream()
-                    .filter(e -> e.getCapabilities().hasCapabilities())
+                    .filter(e -> e.getCapabilities().isEmpty())
```

Comment:
I think here is the ! missing

Reply:
asdf

Reply:
This is a bug which has been introduced with the previous refactoring.

The goal is a list of peers which have no capabilities assigned. This list is counted and then reported either as `log.warn` (in case more than hardcoded 100 persisted peers have no capabilities assigned) or a `log.info`. Currently, however, the code counts peers which have capabilities (`hasCapabilities`) assigned. So no breaking bug that shows itself easily.

This change restores the functionality as intended originally by leaving only peers with empty capability sets in the list.

Reply:
Ah ok. I did not look closer to the intended behaviour...